packages/website/components/tooltip.js

```diff
@@ -0,0 +1,32 @@
+import RCTooltip from 'rc-tooltip'
+import 'rc-tooltip/assets/bootstrap_white.css'
+
+/**
+ * @param {Object} props
+ * @param {JSX.Element} props.children
+ * @param {JSX.Element} props.overlay
+ * @param {string} [props.placement]
+ * @param {boolean} [props.destroyTooltipOnHide]
+ * @param {string} [props.overlayClassName]
+ */
+const Tooltip = ({
+  children,
+  placement = 'top',
+  overlay,
+  destroyTooltipOnHide,
+  overlayClassName = '',
+  ...props
+}) => (
+  <RCTooltip
+    placement={placement}
+    overlay={<div className="bg-white p2 f6 flex">{overlay}</div>}
```

Comment:
I would set a default overlay, but allow an overlay to be passed into this via props as a component.
Right now you just have overlay content, instead, make it the full overlay component.

pseudocode
```
/*
props.overlay= (<div className="bg-white p2 f6 flex">Some Text Here</div>)
*/

overlay={props.overlay}
```

Reply:
This is a refactor request, we can do this another time.